tests/test_routes.py

```diff
@@ -108,4 +108,21 @@ def test_get_not_found(self):
         """It should not Read a Recommendation that is not found"""
         # Get a Recommendation that does not exist
         resp = self.client.get(f"{BASE_URL}/0")
-        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
\ No newline at end of file
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_delete(self):
```

Comment:
did you delete the created recommendation for testing?

Reply:
It seems there is a problem when I do merging. Let me check it again